src/vivarium_public_health/metrics/disease.py

```diff
@@ -55,8 +28,6 @@ class DiseaseObserver:
 
     def __init__(self, disease: str):
         self.disease = disease
-        self.configuration_defaults = self._get_configuration_defaults()
-
         self.disease_component_name = f"disease_model.{self.disease}"
```

Comment:
What is the point of setting the name this way vs having a name property?

Reply:
None. This name doesn't appear to be used. I'll remove.